src/create-config.ts

```diff
@@ -38,8 +38,8 @@ const questions = (choices: Choice[]): PromptObject[] => {
   ];
 };
 
-export const getFormattedTimestamp = () => {
-  return format(new Date(), 'yyMMdd-HHmm');
+export const getFormattedUtcTimestamp = (): string => {
+  return formatInTimeZone(Date.now(), 'UTC', 'yyMMdd-HHmm');
```

Comment:
@Siegrift: WDYT of this?

Reply:
Here: https://github.com/api3dao/operations/blob/43b49dd8caa1f577fe9f12083de1e8ebe6162232/src/create-config.ts#L387
It looks like you fixed the test but not the actual script? hah :stuck_out_tongue_closed_eyes:

Reply:
That can easily be the case 😄 but in this case I left that on purpose and by this comment you made:

> Deployment files are usually keyed by date, that's the only place I can think of off-hand where things are keyed by date.

I guess that line is not wrong after all. The tests are also using the date when reading from `/deployments`

The yyMMdd-HHmm format is only used for secret values like `HEARTBEAT_ID` or `stage` value in config.json